src/main/java/net/pms/newgui/NavigationShareTab.java

```diff
@@ -264,7 +265,7 @@ public void keyReleased(KeyEvent e) {
 
 		// Use MPlayer for video thumbnails
 		mplayerThumb = new JCheckBox(Messages.getString("FoldTab.14"), configuration.isUseMplayerForVideoThumbs());
-		mplayerThumb.setToolTipText(Messages.getString("FoldTab.61"));
+		mplayerThumb.setToolTipText(toolTipMessage("FoldTab.61", "FoldTab.61Note"));
```

Comment:
@ahmedadan why do you not use the `Disabled` here as I introduced? Using this can simplify the message.properties to not translate this general setting for all relevant notes.